src/scss/core/_type.scss

```diff
@@ -0,0 +1,46 @@
+
+h1, .h1 { 
+    font-size: $h1-font-size;
+    line-height: $h1-line-height;
+    font-weight: $font-weight-bold;
+
+    @media (min-width: map-get($grid-breakpoints, lg)) {
+      font-size: $h1-font-size-lg;
+      line-height: $h1-line-height-lg;
+    }
+}
+
+h2, .h2 { 
+    font-size: $h2-font-size; 
+    line-height: $h2-line-height;
+    font-weight: $font-weight-bold;
+
+    @media (min-width: map-get($grid-breakpoints, lg)) {
+      font-size: $h2-font-size-lg;
+      line-height: $h2-line-height-lg;
+    }
+}
+
+h3, .h3 { 
+    font-size: $h3-font-size;
+    line-height: $h3-line-height;
+    font-weight: $font-weight-bold;
+
+    @media (min-width: map-get($grid-breakpoints, lg)) {
+      font-size: $h3-font-size-lg;
+      line-height: $h3-line-height-lg;
+    }
+}
+
+h4, .h4 { 
+    font-size: $h4-font-size;
+    line-height: $h4-line-height;
+    font-weight: $font-weight-normal;
+
+    @media (min-width: map-get($grid-breakpoints, lg)) {
+      font-size: $h4-font-size-lg;
+      line-height: $h4-line-height-lg;
+    }
+}
+
+
```

Comment:
font-size values for mobile are automatically added by bootstrap since we replaced its vars, same for the font-style (to check)